server/chat-plugins/blackjack.ts

```diff
@@ -20,8 +20,6 @@ type SymbolName = 'Hearts' | 'Diamonds' | 'Clubs' | 'Spades';
 export class Blackjack extends Rooms.RoomGame {
 	room: Room;
 
-	blackjack: boolean;
-
 	roomID: RoomID;
```

Comment:
Are these properties necessary? They should already be defined in `RoomGame`, right?

Reply:
In the name of checking later: my response that i sent on discord (roughly)

Possibly? I don't maintain blackjack, so i didn't want to delete anything that wasn't directly relevant to my change.